src/main/java/io/rebloom/client/Client.java

```diff
@@ -336,6 +340,132 @@ public String topkIncrBy(String key, String item, long increment) {
+         SafeEncoder.encode(key), //
+         Protocol.toByteArray(error), //
+         Protocol.toByteArray(probability)).getStatusCodeReply();
+
+     if (!rep.equals("OK")) {
+       throw new JedisException(rep);
+     }
+   }
+ }
+
+ @Override
+ public Long cmsIncrBy(String key, String item, long increment) {
+   try (Jedis conn = _conn()) {
+     return sendCommand(conn, CMSCommand.INCRBY, //
+         SafeEncoder.encode(key), //
+         SafeEncoder.encode(item), //
+         Protocol.toByteArray(increment)).getIntegerMultiBulkReply().get(0);
+   }
+ }
+
+ @Override
+ public List<Long> cmsIncrBy(String key, Map<String, Long> itemIncrements) {
+   try (Jedis conn = _conn()) {
+     List<byte[]> mapFlatten = itemIncrements//
+         .entrySet() //
+         .stream() //
+         .flatMap(e -> Stream.of(//
+             SafeEncoder.encode(e.getKey()), //
+             Protocol.toByteArray(e.getValue())))//
+         .collect(Collectors.toList());
+
+     byte[][] fullArgs = new byte[mapFlatten.size() + 1][];
+     fullArgs[0] = SafeEncoder.encode(key);
+     System.arraycopy(mapFlatten.toArray(), 0, fullArgs, 1, mapFlatten.size());
+
+     return sendCommand(conn, CMSCommand.INCRBY, fullArgs).getIntegerMultiBulkReply();
+   }
+ }
+
+ @Override
+ public List<Long> cmsQuery(String key, String... items) {
+   try (Jedis conn = _conn()) {
+     return sendCommand(conn, key, CMSCommand.QUERY, items).getIntegerMultiBulkReply();
+   }
+ }
+
+ @Override
+ public void cmsMerge(String destKey, String... keys) {
+   try (Jedis conn = _conn()) {
+     byte[][] args = new byte[keys.length + 2][];
+     args[0] = SafeEncoder.encode(destKey);
+     args[1] = Protocol.toByteArray(keys.length);
+     System.arraycopy(SafeEncoder.encodeMany(keys), 0, args, 2, keys.length);
+
+     String rep = sendCommand(conn, CMSCommand.MERGE, args).getStatusCodeReply();
+
+     if (!rep.equals("OK")) {
+       throw new JedisException(rep);
+     }
+   }
+ }
+
+ @Override
+ public void cmsMerge(String destKey, Map<String, Long> keysAndWeights) {
+   try (Jedis conn = _conn()) {
+
+     String[] keys = keysAndWeights.keySet().toArray(new String[0]);
+     String[] weights = keysAndWeights.values().stream().map(l -> l.toString()).collect(Collectors.toList())
+         .toArray(new String[0]);
+
+     byte[][] args = new byte[(keysAndWeights.size() * 2) + 3][];
+
+     args[0] = SafeEncoder.encode(destKey);
+     args[1] = Protocol.toByteArray(keys.length);
+     System.arraycopy(SafeEncoder.encodeMany(keys), 0, args, 2, keys.length);
+     args[keys.length + 2] = SafeEncoder.encode("WEIGHTS");
+     System.arraycopy(SafeEncoder.encodeMany(weights), 0, args, weights.length * 2 + 1, weights.length);
+
+     String rep = sendCommand(conn, CMSCommand.MERGE, args).getStatusCodeReply();
+
+     if (!rep.equals("OK")) {
+       throw new JedisException(rep);
+     }
+   }
+
+ }
+
+ @Override
+ public Map<String, Object> cmsInfo(String key) {
```

Comment:
@bsbodden Can this return Map<String, Long> instead?

Reply:
Nice catch, Kyle! Done. I think other info commands have a mix set of return values but this one only has Longs

Reply:
See https://github.com/RedisBloom/JRedisBloom/pull/29/commits/b55d90748f80ca4aa36dc1516c20309420b1807b